webui/react/src/routes/utils.ts

```diff
@@ -68,14 +68,18 @@ export const handlePath = (
   if (options.onClick) {
     options.onClick(event);
   } else if (href) {
-    if (event.button === 1 || event.metaKey || event.ctrlKey || options.popout) {
+    if (isShiftClick(event) || options.popout) {
       openBlank(href);
     } else {
       routeAll(href);
     }
   }
 };
 
+export const isShiftClick = (event: MouseEvent|React.MouseEvent): boolean => {
+  return event.button === 1 || event.metaKey || event.ctrlKey;
```

Comment:
**nit:** `event.button === 1` is really the middle mouse button (sometimes the clicking of the wheel button).

Reply:
you're telling me that just because is the middle mouse button doesn't match the isShiftClick naming?

Not sure if there's a better name for this method.

Reply:
yeah exactly